extensions/Worker.Extensions.Http.AspNetCore.Analyzers/src/ITypeSymbolExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore
+{
+    internal static class ITypeSymbolExtensions
+    {
```

Comment:
Does this method need to take into account interfaces at all?

Reply:
Given that `OutputBindingAttribute` is not an interface, this will successfully find classes that implement the abstract class. 

For classes that implement some interface, this should just exit (`BaseType` returns `null`).